source/helpers/FileUpload.ts

```diff
@@ -2,19 +2,30 @@ import axios from 'axios';
 import StorageService, { TOKEN_KEY } from '../services/StorageService';
 import { buildServiceUrl } from './UrlHelper';
 
+export const getBlob = async (fileUri: string) => {
+  const response = await fetch(fileUri);
+  const fileBlob = await response.blob();
+  return fileBlob;
+};
+
 /**
  * Helper for uploading a file to S3
  *
  * @param {string} endpoint
  * @param {string} fileName
  * @param {string} fileType
- * @param {obj} fileData should be binary data (a blob, for example)
+ * @param {Blob | Buffer} fileData should be binary data (a blob, for example)
  * @param {obj} headers
  */
-const uploadFile = async (endpoint, fileName, fileType, fileData, headers = {}) => {
+const uploadFile = async (
+  endpoint: string,
+  fileName: string,
+  fileType: string,
+  fileData: Blob | Buffer,
+  headers: Record<string, string> = {}
+) => {
```

Comment:
Replace with object and then match against an interface? :)

Reply:
Yeah sure; five arguments feels like perhaps one or two too many, so I agree.

Reply:
Done.